dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/maintenance/MaintenanceServiceTest.java

```diff
@@ -451,10 +451,11 @@ void testDeleteSoftDeletedEnrollmentLinkedToARelationshipItem()
 
   @Test
   @Disabled("until we can inject dhis.conf property overrides")
-  void testAuditEntryForDeletionOfSoftDeletedTrackedEntity()
-      throws ForbiddenException, NotFoundException, BadRequestException {
+  void testAuditEntryForDeletionOfSoftDeletedTrackedEntity() {
     manager.delete(trackedEntityWithAssociations);
-    assertNull(trackedEntityService.getTrackedEntity(UID.of(trackedEntityWithAssociations)));
+    assertThrows(
+        NotFoundException.class,
```

Comment:
If this is a common case we should add the `Optional<T> find(UID)` method to our service